File: output/ui-acceptance/2026-09-10-e2-expressions/prepare.js
async(page)=>{
 return page.evaluate(async()=>{
  const {ontologyApi:a}=await import('/src/features/semantic/api/ontologyApi.ts');const {useWorkspaceStore:w}=await import('/src/stores/useWorkspaceStore.ts');const ws=w().currentWorkspaceId;
  const imported='Ontology(<urn:e2:lib> Declaration(Class(<urn:e2:Imported>)) AnnotationAssertion(<http://www.w3.org/2000/01/rdf-schema#label> <urn:e2:Imported> "导入定义"@zh))';
  const digest=[...new Uint8Array(await crypto.subtle.digest('SHA-256',new TextEncoder().encode(imported)))].map(x=>x.toString(16).padStart(2,'0')).join('');
  const text=`Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Ontology(<urn:e2:expressions> Import(<urn:e2:lib>)
 Declaration(Class(<urn:e2:Machine>)) Declaration(Class(<urn:e2:Probe>)) Declaration(Class(<urn:e2:Equipment>))
 Declaration(ObjectProperty(<urn:e2:uses>)) Declaration(ObjectProperty(<urn:e2:contains>)) Declaration(ObjectProperty(<urn:e2:related>))
 Declaration(DataProperty(<urn:e2:reading>))
 AnnotationAssertion(rdfs:label <urn:e2:Machine> "机器"@zh)
 AnnotationAssertion(rdfs:label <urn:e2:Machine> "Machine"@en)
 AnnotationAssertion(rdfs:label <urn:e2:Probe> "测针"@zh)
 AnnotationAssertion(rdfs:label <urn:e2:uses> "使用"@zh)
 SubClassOf(<urn:e2:Machine> ObjectIntersectionOf(<urn:e2:Equipment> ObjectSomeValuesFrom(<urn:e2:uses> ObjectUnionOf(<urn:e2:Probe> ObjectComplementOf(<urn:e2:Equipment>)))))
 SubClassOf(<urn:e2:Machine> ObjectAllValuesFrom(<urn:e2:uses> ObjectSomeValuesFrom(<urn:e2:contains> <urn:e2:Probe>)))
 SubClassOf(<urn:e2:Machine> ObjectMinCardinality(2 <urn:e2:uses> <urn:e2:Probe>))
 SubClassOf(<urn:e2:Machine> ObjectMaxCardinality(3 <urn:e2:uses> <urn:e2:Probe>))
 SubClassOf(<urn:e2:Machine> ObjectExactCardinality(2 <urn:e2:uses> <urn:e2:Probe>))
 SubClassOf(<urn:e2:Machine> DataSomeValuesFrom(<urn:e2:reading> xsd:decimal))
 SubClassOf(<urn:e2:Machine> DataAllValuesFrom(<urn:e2:reading> xsd:decimal))
 SubClassOf(<urn:e2:Machine> DataExactCardinality(1 <urn:e2:reading> xsd:decimal))
 SubObjectPropertyOf(ObjectPropertyChain(<urn:e2:uses> <urn:e2:contains> <urn:e2:uses>) <urn:e2:related>)
 HasKey(<urn:e2:Machine> (<urn:e2:uses>) ())
)`;
  const o=await a.create(ws,{name:'E2 复杂表达式验收 '+Date.now(),description:'Isolated synthetic projection fixture'}),d=await a.createDraft(ws,o.id);
  const saved=await a.saveDraft(ws,o.id,{expectedDraftVersion:d.draftVersion,name:o.name,description:o.description,operationId:crypto.randomUUID(),document:{modelSchema:'owl-document-v1',syntax:'FUNCTIONAL',documentText:text,imports:[{requestedIri:'urn:e2:lib',resolvedOntologyIri:'urn:e2:lib',syntax:'FUNCTIONAL',documentText:imported,contentDigest:digest,artifactId:'e2-pinned-library'}],policy:{version:'1',rules:[]}}});
  const report=await a.validate(ws,o.id,saved.draftVersion);if(!report.valid)throw Error(JSON.stringify(report));
  const revision=await a.publish(ws,o.id,{expectedDraftVersion:saved.draftVersion,operationId:crypto.randomUUID(),note:'Synthetic projection acceptance fixture'});
  const draft=await a.createDraft(ws,o.id,revision.id);
  return {ontologyId:o.id,revisionId:revision.id,draftId:draft.id,draftVersion:draft.draftVersion,documentDigest:draft.document.documentDigest,importLockDigest:draft.document.importLockDigest};
 });
}

File: output/ui-acceptance/2026-09-12-t11/prepare.py
"""Isolated lifecycle fixture in existing persistent acceptance database."""
import json,sys,uuid,datetime
from pathlib import Path
OUT=Path(__file__).resolve().parent
sys.path.insert(0,str(OUT.parent/'2026-09-11-b2'))
from runtime_client import Client
c=Client();op=lambda:str(uuid.uuid4())
run=datetime.datetime.now(datetime.timezone.utc).strftime('%Y%m%dT%H%M%SZ')
c.workspace=str(c.call('/workspaces',{'name':'T11 生命周期验收 '+run})['id'])
o=c.call('/semantic/ontologies',{'name':'T11 设备业务模型','description':''});p='/semantic/ontologies/'+o['id']
revs=[]
for name,iri in [('设备','Equipment'),('生产设备','Equipment'),('生产设备','Machine')]:
 d=c.call(p+'/draft',{})
 doc=f'Ontology(<urn:t11:model> Declaration(Class(<urn:t11:{iri}>)) AnnotationAssertion(<http://www.w3.org/2000/01/rdf-schema#label> <urn:t11:{iri}> "{name}") Declaration(DataProperty(<urn:t11:voltage>)) DataPropertyDomain(<urn:t11:voltage> <urn:t11:{iri}>) DataPropertyRange(<urn:t11:voltage> <http://www.w3.org/2001/XMLSchema#decimal>))'
 d=c.call(p+'/draft',{'expectedDraftVersion':d['draftVersion'],'operationId':op(),'name':o['name'],'description':'','document':{'modelSchema':'owl-document-v1','syntax':'FUNCTIONAL','documentText':doc,'imports':[],'policy':{'version':'1','rules':[]}}},'PUT')
 assert c.call(p+'/draft/validate',{'expectedDraftVersion':d['draftVersion']})['valid']
 revs.append(c.call(p+'/draft/publish',{'expectedDraftVersion':d['draftVersion'],'operationId':op(),'note':name+'版本'}))
apps=[]
for label in ['装配车间','维修车间']:
 kb=c.call('/wiki/knowledge-bases',{'name':'T11 '+label})
 b=c.call('/semantic/knowledge-bases/'+str(kb['id'])+'/binding',{'action':'ENABLE','revisionId':revs[0]['id']},'PUT')
 e=c.call('/semantic/graphs/'+b['graphId']+'/entities',{'iri':'urn:t11:equipment:'+str(kb['id']),'displayName':label+'设备A','assertedTypes':['urn:t11:Equipment']})
 apps.append({'kbId':str(kb['id']),'name':'T11 '+label,'graphId':b['graphId'],'entity':e})
d=c.call(p+'/draft',{})
(OUT/'fixture.json').write_text(json.dumps({'workspaceId':c.workspace,'ontologyId':o['id'],'revisions':revs,'applications':apps,'draftId':d['id']},ensure_ascii=False,indent=2))
print('Created T11 isolated fixture '+o['id'])
